info.go

```diff
@@ -259,6 +262,15 @@ func (pi *ProgramInfo) Runtime() (time.Duration, bool) {
 	return time.Duration(0), false
 }
 
+// RecursionMisses returns the total number of times the program was NOT called.
+//
+// It is INDEPENDENT on whether collection of statistics is enabled or not, but
```

Comment:
I'd remove the caps and make the sentence more straightforward:
```suggestion
// This value is always measured on supported kernels, regardless of statistics being enabled or not.
```

Also drop the part about the bool as we're discussing below. If the bool is always true and a real probe is not planned, we may as well drop it.